tests/integrationtest/t/executor/simple.test

```diff
@@ -379,6 +379,11 @@ create user user1;
 ALTER USER `user1` RESOURCE GROUP `rg1`;
 SELECT CURRENT_RESOURCE_GROUP();
 
+create role role_for_resource_group;
```

Comment:
Please add a simple `create role role_for_resource_group resource group rg1;` to cover `s.IsCreateRole` branch in `executeCreateUser`

Reply:
[`CREATE ROLE`](https://docs.pingcap.com/zh/tidb/stable/sql-statement-create-role) syntax doesn't support setting the resource group, so it will directly return an error when parsing the SQL.

<img width="1480" alt="image" src="https://github.com/pingcap/tidb/assets/1446531/a09d3774-bdd0-4985-b300-74ef56fc7fa5">